docs/features/kubernetes/authentication.md

```diff
@@ -55,6 +55,37 @@ To get the API server address for your Azure cluster, go to the Azure console pa
 cluster resource, go to `Overview` > `Properties` tab > `Networking` section and copy paste
 the API server address directly in that `url` field.
 
+### AWS
+
+The Amazon Web Services (AWS) server side authentication provider works by connecting to [AWS IAM (Identity and Access Management)][3], which defines who (users) and what (workloads) can access which AWS resources. AWS IAM users can be granted an _Access Key ID_ and _Secret Access Key_ for authentication.
+
+In order for Backstage to read Kubernetes clusters running in AWS [Elastic Kubernetes Service (EKS)][4], your chosen IAM user must also have authorization to view EKS clusters. To do this, create or attach an [EKS IAM policy][5] to your IAM user before proceeding.
+
+To configure the AWS auth provider with the Kubernetes plugin, add the following to the `auth` section of your Backstage configuration:
+
+```yaml
+kubernetes:
+  serviceLocatorMethod:
+    type: 'multiTenant'
+  clusterLocatorMethods:
+    - type: 'config'
+      clusters:
+        - url: https://<cluster-name>.<region>.eks.amazonaws.com
+          name: <cluster-name>
+          authProvider: 'aws'
+          skipTLSVerify: true
+          skipMetricsLookup: true
+auth:
+  environment: 'production'
+  providers:
+    aws:
+      development:
+        accessKeyId: ${AWS_ACCESS_KEY_ID}
+        secretAccessKey: ${AWS_SECRET_ACCESS_KEY}
```

Comment:
Is this supposed to be under `aws.accounts` or `aws.mainAccount` instead? per https://github.com/backstage/backstage/blob/master/packages/integration-aws-node/README.md#backstage-app-configuration

As far as I see this config path doesn't exist